recipes/specta/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "specta" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: eebf64170e918831e75b99d9bb15cacab0a6a028f2eb3f8587fe81ab7fcc1a1b
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  build:
+    - nodejs
```

Comment:
This can be in host. Build is a special section for compilers and cross-compiling stuff. This is noarch and it should be OK to have everything in host.